domain/domain.go

```diff
@@ -23,7 +23,7 @@ import (
 	"unsafe"
 
 	"github.com/coreos/etcd/clientv3"
-	"github.com/grpc-ecosystem/go-grpc-prometheus"
+	grpc_prometheus "github.com/grpc-ecosystem/go-grpc-prometheus"
```

Comment:
I'm confused in this line.
This line changed because of my code style plugin.
I found in `domain.go`, it used by `grpc_prometheus.XXX()`.

Reply:
We don't change it.